march_joint_inertia_controller/package.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>march_joint_inertia_controller</name>
+  <version>0.0.0</version>
+  <description>The march_joint_inertia_controller package</description>
```

Comment:
It's better to create a more detailed description of the package functionality (look at other packages as examples)
```suggestion
  <description>This package creates a ROS controller which can be customized to operate in different circumstances.  </description>
```